napari-workshops/notebooks/spot_detection_basic.md

```diff
@@ -0,0 +1,270 @@
+Let's hide the `low_pass` and take a screenshot for documentation.
+
+```{code-cell} ipython3
+viewer.layers['low_pass'].visible = False
+```
+
+```{code-cell} ipython3
+nbscreenshot(viewer)
+```
+
+## Detect spots
+
+Next, we will use one of the blob detection algorithms from sci-kit image to
+perform the blob detection.
+
+```{tip}
+- See the [blob detection tutorial from scikit-image](https://scikit-image.org/docs/stable/auto_examples/features_detection/plot_blob.html). We recommend the [blob_log detector](https://scikit-image.org/docs/stable/api/skimage.feature.html#skimage.feature.blob_log), but feel free to experiment!
+- See the "Note" from the blob_log docs: "The radius of each blob is approximately $\sqrt{2}\sigma$ for a 2-D image"
+```
+
+```{code-cell} ipython3
+from skimage.feature import blob_log
+
+# detect the spots on the filtered image
+blobs_log = blob_log(
+    high_passed_spots,
+    max_sigma=3,
+    threshold=None, # use a relative threshold instead
+    threshold_rel=0.2
+)
+    
+# convert the output of the blob detector to the 
+# desired points_coords and sizes arrays
+# (see the docstring for details)
+spot_coords = blobs_log[:, 0:2]
+spot_sizes = 2 * np.sqrt(2) * blobs_log[:, 2]
+```
+
+To visualize the results, add the spots to the viewer as a
+[Points layer](https://napari.org/stable/tutorials/fundamentals/points.html). If you
+would like to see an example of using a points layer, see
+[this example](https://napari.org/gallery/add_points.html).
+
+```{code-cell} ipython3
+# add the detected spots to the viewer as a Points layer
+viewer.add_points(spot_coords, size=spot_sizes)
+```
+
+```{code-cell} ipython3
+nbscreenshot(viewer)
+```
+
+Let's zoom in for a better look. You can explore interactively in the viewer and then you can explicitly
+set the center of the field of view and the zoom factor.
+
+```{code-cell} ipython3
+viewer.camera.center = (200, 270)
+viewer.camera.zoom = 8
+```
+
+```{code-cell} ipython3
+nbscreenshot(viewer)
+```
+
+## Optional: using Points layer `properties`
+
+`properties` is a table associated with a Points layer that can store additional data associated with
+a data point. It has one row per data element (Point) and one column per feature. Importantly,
+napari can not only display the values of associated features in the status bar, but also use them for
+styling, e.g. for `face_color`. For more information, see the [Points layer guide](https://napari.org/stable/howtos/layers/points.html#setting-point-edge-and-face-color-with-properties).
+
+```{note}
+napari plans to deprecate `properties` in favor of `features` in a future release, but for the time being
+the documentation for `properties` is superior, so we use that here. Note that some examples in the Gallery
+have been updated to use `features`, but for the purposes of these simple examples you can replace
+`features` with `properties` or vice versa.
+```
```

Comment:
This is pretty oof. 😣 Is there anything more than a find/replace on the docs to fix this? I could insta-merge and then we could use features here and get rid of this warning...

Reply:
This comment might be out-dated. I've not done full due-diligence. I think the example and tutorial are migrated by Andy.

Reply:
OK, I've moved everything to using features and changed the copy here, removing the note.